Makefile

```diff
@@ -11,7 +11,12 @@ setup-git:
 	pre-commit install --install-hooks
 
 test:
-	SNUBA_SETTINGS=test pytest -vv -v -m "not ci_only"
+	SNUBA_SETTINGS=test pytest -vv tests -v -m "not ci_only"
+
+test-distributed-migrations:
+	docker-compose -f docker-compose.gcb.clusters.yml down
+	SNUBA_IMAGE=snuba-test SNUBA_SETTINGS=test docker-compose -f docker-compose.gcb.clusters.yml run --rm snuba-test
```

Comment:
im assuming `SNUBA_SETTINGS` here should also be `test_distributed_migrations` yeah?

Reply:
fixed